### ISSUE TITLE
Bug: Warning about config issues recommends running :LualineNotices, but that spawns an empty window is back?

### ISSUE THREAD

**Issue by mgrubb** [nvim-lualine/lualine.nvim#693]
This is a continuation of Issue #632 that was closed.

### Self Checks

- [x] I'm using the latest lualine.
- [x] I didn't find the issue in exsisting issues or prs. (closed issue #632 did not resolve this.)

### How to reproduce the problem
Using minimal config below, and running PackerSync reproduces error.

### Expected behaviour
Not receive warning about config issues

### Actual behaviour
Receiving warning about config issues

### Aditional information
I have tried everything I know to make sure that colorscheme is set before lualine is configured.
I've tried putting the colorscheme package in its own `use` block.  The only thing that solves this problem is not setting a colorscheme at all.  I think I've tried every permutation that I could think of to get the colorscheme set before lualine and I'm still getting this behavior.

```lua
require('packer').startup(function()
  -- Base functionality
  use 'wbthomason/packer.nvim' 
  use { 'nvim-lualine/lualine.nvim',
        opt = false,
        requires = {
          { 'kyazdani42/nvim-web-devicons', opt = false },
          { 'chriskempson/base16-vim', opt = false },
        },
        config = function()
          local function diff_source()
            local gitsigns = vim.b.gitsigns_status_dict
            if gitsigns then
              return {
                added = gitsigns.added,
                modified = gitsigns.changed,
                removed = gitsigns.removed,
              }
            end
          end

          vim.cmd [[colorscheme base16-outrun-dark]]

          require('lualine').setup({
            options = {
              theme = 'auto',
              icons_enabled = true,
              disabled_filetypes = {},
              always_divide_middle = false,
            },
            sections = {
              lualine_a = { {"b:gitsigns_head", icon = "" }, { "diff", soruce = diff_source } },
              lualine_b = {
                {
                  'diagnostics',
                  sources = { 'nvim_diagnostic' },
                  sections = { 'error', 'warn', 'info', 'hint' },
                },
              },
              lualine_c = {
                {
                  'filetype',
                  icon_only = true,
                },
                {
                  'filename',
                  file_status = true,
                  path = 1,
                  shorting_target = 40,
                  symbols = { modified = "[]", readonly = " " },
                },
              },
              lualine_x = {
                'encoding',
                'fileformat',
                'filesize',
              },
              lualine_y = { 'progress' },
              lualine_z = { 'location' },
            },
            inactive_sections = {
              lualine_a = {},
              lualine_b = {},
              lualine_c = { 'filename' },
              lualine_x = { 'location' },
              lualine_y = {},
              lualine_z = {},
            },
            tabline = {},
            extensions = { 'nvim-tree', 'toggleterm', 'quickfix', 'symbols-outline' },
          })
        end,
        after = 'base16-vim',
      }
end)
```

**Comment by shadmansaleh** [comment]
try the patch from [here](https://github.com/nvim-lualine/lualine.nvim/issues/632#issuecomment-1085620986) it should help us figure out what notice you were actually suppose to see.

**Comment by mgrubb** [comment]
```
stack traceback:
^I...ck/packer/opt/lualine.nvim/lua/lualine/utils/notices.lua:12: in function 'add_notice'
^I...ck/packer/opt/lualine.nvim/lua/lualine/themes/base16.lua:4: in function 'add_notice'
^I...ck/packer/opt/lualine.nvim/lua/lualine/themes/base16.lua:57: in function 'setup_base16'
^I...ck/packer/opt/lualine.nvim/lua/lualine/themes/base16.lua:92: in main chunk
^I[C]: in function 'dofile'
^I...ack/packer/opt/lualine.nvim/lua/lualine/utils/loader.lua:228: in function <...ack/packer/opt/lualine.nvim/lua/lualine/utils/loader
.lua:213>
^I[C]: in function 'pcall'
^I...pack/packer/opt/lualine.nvim/lua/lualine/themes/auto.lua:15: in main chunk
^I[C]: in function 'dofile'
^I...ack/packer/opt/lualine.nvim/lua/lualine/utils/loader.lua:228: in function <...ack/packer/opt/lualine.nvim/lua/lualine/utils/loader
.lua:213>
^I[C]: in function 'pcall'
^I...e/nvim/site/pack/packer/opt/lualine.nvim/lua/lualine.lua:231: in function 'get_theme_from_config'
^I...e/nvim/site/pack/packer/opt/lualine.nvim/lua/lualine.lua:249: in function 'setup_theme'
^I...e/nvim/site/pack/packer/opt/lualine.nvim/lua/lualine.lua:330: in function 'setup'
^I[string "..."]: in function <[string "..."]:0>
^I[C]: in function 'pcall'
^I/Users/mgrubb/.config/nvim/plugin/packer_compiled.lua:60: in function 'try_loadstring'
^I/Users/mgrubb/.config/nvim/plugin/packer_compiled.lua:111: in function </Users/mgrubb/.config/nvim/plugin/packer_compiled.lua:10>
^I[C]: in function 'pcall'
^I/Users/mgrubb/.config/nvim/plugin/packer_compiled.lua:10: in main chunk
```

**Comment by mgrubb** [comment]
So I've managed to resolve this by switching from chriskempson/base16-vim to RRethy/nvim-base16.
Is there a way for lualine to be more agnostic about themes and query the current theme for colors, instead of having to implicitly support different themes?  Not sure if that is feasible or even possible.

**Comment by abhishek-1131** [comment]
> So I've managed to resolve this by switching from chriskempson/base16-vim to RRethy/nvim-base16. Is there a way for lualine to be more agnostic about themes and query the current theme for colors, instead of having to implicitly support different themes? Not sure if that is feasible or even possible.

Hi, this did not work for me. How did you make the switch from the first theme to the second? 
In my case, I did `use({"nvim-lualine/lualine.nvim", requires = {'RRethy/nvim-base16'}})` in my packer plugins config, but the issue persists.

**Comment by troyanov** [comment]
Hi @abhishek-1131 
I faced the same issue last night and I've discovered that problems started after f3dc0dcd4c0c75ae46ea8d6bccd715c5107f22fd because when I used c8e5a69085e89c2bac6bd01c74fcb98f9ffa5cdc there were no issues.

After looking at the diff I realised that it might be something related to the `runtimepath` 
Indeed, long time ago I transitioned from vim following this [tutorial] and made my runtime path being `~/.vim`
(https://neovim.io/doc/user/nvim.html#nvim-from-vim)
```
1. To start the transition, create your |[init.vim](https://neovim.io/doc/user/init.html)| (user [config](https://neovim.io/doc/user/starting.html#config)) file:

    :call mkdir(stdpath('config'), 'p')
    :exe 'edit '.stdpath('config').'/init.vim'

2. Add these contents to the file:

    set runtimepath^=~/.vim runtimepath+=~/.vim/after
    let &packpath = &runtimepath
    source ~/.vimrc
```

Maybe you also have the same thing in your `init.vim`?

**Comment by shadmansaleh** [comment]
fixed in aeeefcef98ce48315deaa274a19a9847a1753b8e
